cv-basics/5_opencv_overview.ipynb

```diff
@@ -143,7 +143,8 @@
      "name": "stdout",
```

Comment:
<p>Should there be error message here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>No actually this was executed before executing above cells , hence the error. Syntactically its sound.</p>

Reply:
Fixed the output error message.